### ISSUE TITLE
pip install --"unstructured[all-docs]" : there is a mistake with this command to be corrected to: pip install --q "unstructured[all-docs]" 

### ISSUE THREAD

**Issue by bchatelet** [tonykipkemboi/ollama_pdf_rag#2]
**Describe the bug**

**To Reproduce**

**Expected behavior**

**Screenshots**

**Desktop (please complete the following information):**

**Smartphone (please complete the following information):**

**Additional context**

**Comment by bchatelet** [comment]
To be corrected to:
 pip install --q "unstructured[all-docs]"

**Comment by ozwiz** [comment]
I tried to run the notebook in Google Colab.
I edited the first cell and changed the second line as pip install --q "unstructured[all-docs]".

Then I got the following error:

ERROR: pip's dependency resolver does not currently take into account all the packages that are installed. This behaviour is the source of the following dependency conflicts.
imageio 2.31.6 requires pillow<10.1.0,>=8.3.2, but you have pillow 10.3.0 which is incompatible.
